packages/web-twig/src/Resources/components/Field/__tests__/__fixtures__/validationText.twig

```diff
@@ -1,22 +1,25 @@
 <!-- DO NOT render when there is nothing to display -->
-<ValidationText className="myClass" />
+<ValidationText className="myClass" id="validationTextNoValidation" />
```

Comment:
The IDs should be in the kebab-case. I will fix this.

Reply:
You are right. Damn you copilot :D